### PR TITLE
Add workload field

### LINKED ISSUE #76: Add workload field
Allow users to rate tasks in terms of how much effort may be required.

### PR DESCRIPTION
**Pull request by yhtMinceraft1010X**
Fixes #76

### CODE REVIEW

**Review by AiwassPrime — COMMENTED** [review]
Tests are well implemented. Great work!

File: src/test/java/seedu/module/model/task/WorkloadTest.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        Workload firstWorkload = new Workload("1");
+        Workload secondWorkload = new Workload("1");
+        Workload thirdWorkload = new Workload("2");
+        String somethingElse = "Not Workload";
+
+        assertTrue(firstWorkload.equals(firstWorkload));
+        assertTrue(firstWorkload.equals(secondWorkload));
+        assertFalse(firstWorkload.equals(thirdWorkload));
+        assertFalse(firstWorkload.equals(somethingElse));
+    }
```
Comment (figo2127) [inline src/test/java/seedu/module/model/task/WorkloadTest.java:60]: Looking neat!

**Review by figo2127 — COMMENTED** [review]
looks good

File: src/main/java/seedu/module/logic/commands/EditCommand.java [hunk 2]
```diff
@@ -187,6 +193,14 @@ public void setDescription(Description description) {
...
         }
 
+        public void setWorkload(Workload workload) {
+            this.workload = workload;
+        }
+
+        public Optional<Workload> getWorkload() {
+            return Optional.ofNullable(workload);
+        }
+
```
Comment (umergta) [inline src/main/java/seedu/module/logic/commands/EditCommand.java:203]: nice!

File: src/main/java/seedu/module/model/task/Workload.java [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+import java.util.InputMismatchException;
+
+/**
+ * Enum to set workload levels of tasks.
+ */
+public class Workload {
+    public static final String LOW_WORKLOAD = "Low";
+    public static final String MEDIUM_WORKLOAD = "Medium";
+    public static final String HIGH_WORKLOAD = "High";
```
Comment (umergta) [inline src/main/java/seedu/module/model/task/Workload.java:14]: can we use enum? just a suggestion

**Review by umergta — COMMENTED** [review]
LGTM :)

File: src/main/java/seedu/module/model/task/Workload.java (hunk 3 above)
Comment (yhtMinceraft1010X) [reply src/main/java/seedu/module/model/task/Workload.java:14]: I was considering using enum but I couldn't get the constructors to work. Maybe I'll look into it in v1.3.

**Review by QY-H00 — COMMENTED** [review]
LGTM for the testing part.

**Pull request merged**